godel/config/godel.properties

```diff
@@ -1,2 +1,2 @@
-distributionURL=https://palantir.bintray.com/releases/com/palantir/godel/godel/2.8.0/godel-2.8.0.tgz
-distributionSHA256=edaeb3676c6a7dee867d91c1013d0d1f503a4ab5d444d01c422c131bbeae876a
+distributionURL=https://palantir.bintray.com/releases/com/palantir/godel/godel/2.9.0/godel-2.9.0.tgz
+distributionSHA256=
```

Comment:
Would recommend setting the checksum here (this one can just be copied from Bintray :).

Value is `c00669bd13e4f3ee28833ca115a25bb2f965b7a6b55a66c4adb17eac90bd5280` as listed on https://bintray.com/palantir/releases/godel